File: client1.py
import socket

HOST = 'localhost'
PORT = 5005

client = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
client.connect((HOST, PORT))

while True:
    message = input("You: ")
    client.send(message.encode())

    if message.lower() == "stop":
        break

    data = client.recv(1024).decode()
    print("Server:", data)

client.close()

File: Client 1.java
import java.io.DataInputStream;
import java.io.DataOutputStream;
import java.net.Socket;
import java.io.BufferedReader;
import java.io.InputStreamReader;

public class Client {
    public static void main(String[] args) {
        try {
            Socket s = new Socket("localhost", 5004);
            DataInputStream din = new DataInputStream(s.getInputStream());
            DataOutputStream dout = new DataOutputStream(s.getOutputStream());
            BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
            String str = " ", str2 = " ";
            while (!str.equals("stop")) {
                str = br.readLine();
                dout.writeUTF(str);
                dout.flush();
                str2 = (String)din.readUTF();
                System.out.println("Server: "+str2);
            }
            din.close();
            dout.close();
            s.close();
        } catch (Exception e) {
            System.out.println(e);
        }
    }
}
